grpc/src/test/java/com/linecorp/armeria/client/grpc/GrpcClientBuilderTest.java

```diff
@@ -154,4 +172,50 @@ public <I, O> ClientCall<I, O> interceptCall(MethodDescriptor<I, O> method,
         assertThat(clientParams.options().get(GrpcClientOptions.INTERCEPTORS))
                 .containsExactly(interceptorA, interceptorB);
     }
+
+    @RegisterExtension
+    static ServerExtension server = new ServerExtension() {
+        @Override
+        protected void configure(ServerBuilder sb) {
+            sb.service(
+                    GrpcService.builder()
+                               .addService(new TestServiceImpl(Executors.newSingleThreadScheduledExecutor()))
+                               .build());
+        }
+    };
+
+    @Test
+    void useDefaultGrpcExceptionHandlerFunction() {
+        final TestServiceBlockingStub client = GrpcClients.builder(server.httpUri())
+                                                          .build(TestServiceBlockingStub.class);
+
+        final ClientBuilderParams clientParams = Clients.unwrap(client, ClientBuilderParams.class);
+        assertThat(clientParams.options().get(GrpcClientOptions.EXCEPTION_HANDLER))
+                .isEqualTo(GrpcExceptionHandlerFunction.ofDefault());
```

Comment:
`isSameAs()` is probably a better choice in this case because we want it to be an exactly same instance.

Reply:
This testcase has been removed after some modification.
But there is still test case `useDefaultGrpcExceptionHandlerFunctionAsFallback`